packages/wxt/src/core/builders/vite/plugins/wxtPluginLoader.ts

```diff
@@ -63,7 +80,7 @@ try {
           const newHead = document.createElement('head');
           document.documentElement.prepend(newHead);
         }
-        document.head.prepend(script);
+        document.head?.prepend(script);
```

Comment:
nitpick `document.head` exists at this point. Don't have to perform optional chaining

Reply:
I've wondering why check looks like `document.head == null` and there's a problem i've changed it to `===` and now optional chaining isn't necessary, i want to do it before, but i wasn't sure why somebody want to check also `null` as string, i think it's mistake, which was resolved now :)

Reply:
Uh that's probably because, of some JS magic. `a == null` is true, if a is either null or undefined so maybe that's the reason. This change might break it so better consult with the owner before this

Reply:
> Uh that's probably because, of some JS magic. `a == null` is true, if a is either null or undefined so maybe that's the reason. This change might break it so better consult with the owner before this
> 
> 

Tests passed, but overall owner need to take a look on these changes :)